brd-ios/breadwallet/Assets.xcassets/avatars/earth.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "earth.pdf",
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  },
+  "properties" : {
+    "template-rendering-intent" : "original"
+  }
+}
```

Comment:
You might need to activate "Preserve Vector Data" for all the PDF assets you are adding.

Reply:
why?

Reply:
https://useyourloaf.com/blog/xcode-9-vector-images/
https://www.hackingwithswift.com/example-code/xcode/how-to-use-vector-images-in-your-asset-catalog

"The other is Preserve Vector Data, which copies the PDF into the final build so it can be re-rasterized at runtime to fit any size need"
"Xcode 9 changes this situation by allowing you to choose to treat the image as a vector at runtime. This means the image can smoothly scale to arbitrary sizes."